infrastructure/tari_script/src/script.rs

```diff
@@ -505,6 +505,22 @@ impl TariScript {
         }
     }
 
+    /// Validates an m-of-n multisig script
+    ///
+    /// This validation broadly proceeds to check if **exactly** _m_ signatures are valid signatures out of a
+    /// possible _n_ public keys.
+    ///
+    /// A successful validation returns `Ok(P)` where _P_ is the sum of the public keys that matched the _m_
+    /// signatures. If the validation was NOT successful, `check_multisig` returns `Ok(None)`. This is a private
+    /// function, and callers will interpret these results according to their use cases.
+    ///
+    /// Other problems, such as stack underflows, invalid parameters etc return an `Err` as usual.
+    ///
+    /// Notes:
+    /// * The _m_ signatures are expected to be the top _m_ items on the stack.
+    /// * Every public key can be used AT MOST once.
+    /// * Every signature MUST be a valid signature using one of the public keys
+    /// * _n_ and _m_ must be positive AND n <= m AND n <= MAX_MULTISIG_LIMIT (32).
```

Comment:
```suggestion
    /// * _m_ and _n_ must be positive AND m <= n AND n <= MAX_MULTISIG_LIMIT (32).
```

Reply:
from what I can see (and recall) in the code anyway :P

Reply:
derp